controllers/managedocs_controller.go

```diff
@@ -68,6 +69,11 @@ const (
 	managedOCSName                         = "managedocs"
 	storageClusterName                     = "ocs-storagecluster"
 	prometheusName                         = "managed-ocs-prometheus"
+	prometheusServiceName                  = "managed-ocs-prometheus-service"
+	servingCertSecretBetaAnnotationKey     = "service.beta.openshift.io/serving-cert-secret-name"
+	servingCertSecretAlphaAnnotationKey    = "service.alpha.openshift.io/serving-cert-secret-name"
```

Comment:
- I don't think we also need alpha key

Reply:
There was a discussion on old api-s and backward compatibilities. @nb-ohad , can you comment on this

Reply:
Please keep both the alpha and beta key. There is no harm in doing so and we get better coverage